cilium/cilium.io
# refactor: update home page

### Describe changes

This PR brings an updated version of the `Home` page based on [the design](https://www.figma.com/file/70psWC4KMJmUsQ38noybaw/Cilium-Website-Design?node-id=3110%3A7732)

### Steps to test:
1. Go to [/home](https://deploy-preview-51--cilium.netlify.app/) page
2. Confirm everything looks as expected

### Important points:
-  This PR doesn't have the updated header&footer, as they are brought by another PR
- This PR requires an update in Hero illustration, it will be updated as soon as it is ready
- The Discover section is in its old look at the moment, but the updated one (with the heading **Get Hands-On With Cilium** ) will be added with the PR #31 

### TODO:
- [ ] add updated Hero illustration when it's ready
